perllib/FixMyStreet/Cobrand/Hounslow.pm

```diff
@@ -6,7 +6,7 @@ use warnings;
 
 sub council_area_id { 2483 }
 sub council_area { 'Hounslow' }
-sub council_name { 'Hounslow Borough Council' }
+sub council_name { 'Hounslow Highways' }
```

Comment:
I wonder if at some point we want a display name and a body name but probably not worth it unless we get more of these.